versioncontrol/peer.go

```diff
@@ -17,15 +19,24 @@ import (
 	"storj.io/storj/internal/version"
 )
 
-// Config is all the configuration parameters for a Version Control Server
+var (
+	// RolloutErr defines the rollout config error class.
+	RolloutErr = errs.Class("rollout config error")
+	// SeedLength is the number of bytes in a rollout seed.
+	SeedLength = 32
+
+	hexLenFactor = 2
```

Comment:
Shouldn't `SeedLength` and `hexLenFactor` be constants instead of variables?

Reply:
indeed